testsuite/MDAnalysisTests/converters/test_parmed_parser.py

```diff
@@ -121,20 +159,47 @@ class TestParmedParserPSF(BaseTestParmedParser):
     expected_n_cmaps = 212
     elems_ranges = ((100, 120),)
     # No atomic numbers set by parmed == no elements
-    expected_elems = (np.array(
-        ['N', 'H', 'C', 'H', 'C', 'H', 'H', 'C', 'H', 'C', 'H', 'H', 'H', 'C',
-         'H', 'H', 'H', 'C', 'O', 'N',], dtype=object),)
+    expected_elems = (
+        np.array(
+            [
+                "N",
+                "H",
+                "C",
+                "H",
+                "C",
+                "H",
+                "H",
+                "C",
+                "H",
+                "C",
+                "H",
+                "H",
+                "H",
+                "C",
+                "H",
+                "H",
+                "H",
+                "C",
+                "O",
+                "N",
+            ],
```

Comment:
I don't mind, but maybe someone does.

Reply:
me neither. Could be `np.array(list('NHCHCHHCHCHHHCHHHCON'))` but that's another story I guess

Reply:
That is actually a very good suggestion. If there are no objections, I'll change it to that.